projects/eHealthSuisse/eMedikation/v1.0/2-4-MedicationDispense.xml

```diff
@@ -295,67 +221,56 @@
 									</reference>
 								</substanceAdministration>
 							</entryRelationship>
-							<!-- dosage insruction -->
-							<entryRelationship typeCode='COMP'>
-								<substanceAdministration classCode='SBADM'
-									moodCode='INT'>
-									<templateId root='1.3.6.1.4.1.19376.1.9.1.3.6' />
-
-									<!-- split dosing -->
+							<entryRelationship typeCode="COMP">
+								<substanceAdministration classCode="SBADM" moodCode="INT">
+									<templateId root="1.3.6.1.4.1.19376.1.9.1.3.6" />
 									<templateId root="1.3.6.1.4.1.19376.1.5.3.1.4.9" />
-
 									<effectiveTime xsi:type="IVL_TS">
 										<low value="20120204" />
+										<high nullFlavor="UNK" />
 									</effectiveTime>
-									<repeatNumber value="7" />
-									<routeCode code="20053000" codeSystem="0.4.0.127.0.16.1.1.2.1"
-										displayName="Oral use" />
+									<routeCode code="20053000" codeSystem="0.4.0.127.0.16.1.1.2.1" displayName="Oral use" />

```

Comment:
This currently does not pass validation because of the missing repeatNumber. It's an error on my side, I've updated https://art-decor.org/art-decor/decor-issues--ch-pharm-?id=2.16.756.5.30.1.127.77.4.6.6